crates/oxc_minifier/tests/peephole/mod.rs

```diff
@@ -22,7 +22,7 @@ fn integration() {
     return console.log(JSON.stringify(os))
     })",
         r#"require("./index.js")(function(e, os) {
-    return console.log(e || JSON.stringify(os));
+    return e ? console.log(e) : console.log(JSON.stringify(os));
```

Comment:
how come this changed? `e` isn't a global so shouldn't have a side effect right?

Reply:
It's not the case here, but `console.log` may change `e`.
```js
var console = { get log() { e = false; return () => {} } }
return e ? console.log(e) : console.log(JSON.stringify(os));
// this cannot be compressed to `return console.log(e ? e : JSON.stringify(os));`
```

Reply:
ohhh gotcha, thanks for explaining.